cinode-recruitment.php

```diff
@@ -163,6 +163,10 @@ function cinode_recruitment_route()
 						'type' => 'int',
 						'description' => 'campaignCode',
 					),
+					'availableFrom' => array(
+						'type' => 'int',
```

Comment:
I don't speak php, just double checking that the `int` type is correct here.

Reply:
This is how its defined WordPress service API, tested with int and string, its working fine with both with cinode API